vehicle/delete.go

```diff
@@ -20,5 +22,25 @@ func NewDeleteHandler(store storage.Store, logger *zap.Logger) *DeleteHandler {
 }
 
 func (d *DeleteHandler) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
-	http.Error(rw, "Not Implemented", http.StatusInternalServerError)
+	strId := r.PathValue("id")
+
+	id, err := strconv.ParseInt(strId, 10, 64)
+
+	if err != nil {
+		fmt.Println("An error has occured", err)
+		rw.WriteHeader(http.StatusNotFound)
```

Comment:
j'avais fait la même chose et le prof m'a expliqué que ça servait à R. CF documentation

Reply:
pardon j'avais pas vu ! c'est plutot : http.Error(rw, "Failed to parse ID", http.StatusInternalServerError) pour la gestion des erreurs